src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -30,5 +36,67 @@ public IEnumerable<Assembly> RetrievePluginAssemblies(IEnumerable<Assembly> defa
 
             return assemblies;
         }
+
+        protected virtual void ProcessBaseline(IAnalysisContext context, T driverOptions, IFileSystem fileSystem)
+        {
+            if (!(driverOptions is AnalyzeOptionsBase options))
+            {
+                return;
+            }
+
+            if (string.IsNullOrEmpty(options.BaselineSarifFile) || string.IsNullOrEmpty(options.OutputFilePath))
+            {
+                return;
+            }
+
+            var serializer = new JsonSerializer
+            {
+                Formatting = options.PrettyPrint || (!options.PrettyPrint && !options.Minify) ?
+                                    Formatting.Indented :
+                                    Formatting.None
+            };
+
+            SarifLog baselineFile;
+            using (var reader = new JsonTextReader(new StreamReader(fileSystem.FileOpenRead(options.BaselineSarifFile))))
+            {
+                baselineFile = serializer.Deserialize<SarifLog>(reader);
+            }
```

Comment:
you can replace this by:
```
baselineFile = SarifLog.Load(optons.BaselineSarifFile)
```

Reply:
Can I inject a mock IFileSystem to SarifLog for test cases?